src/bundle/Resources/public/js/scripts/sidebar/btn/location.edit.js

```diff
@@ -14,17 +15,31 @@
         const showModal = (modalHtml) => {
             const wrapper = doc.querySelector('.ez-modal-wrapper');
             wrapper.innerHTML = modalHtml;
-            wrapper.querySelector('.ez-btn--add-draft').addEventListener('click', addDraft, false);
+            const addDraftButton = wrapper.querySelector('.ez-btn--add-draft');
+            if (addDraftButton) {
+                addDraftButton.addEventListener('click', addDraft, false);
+            }
             [...wrapper.querySelectorAll('.ez-btn--prevented')].forEach(btn => btn.addEventListener('click', event => event.preventDefault(), false));
             $('#version-draft-conflict-modal').modal('show').on('hidden.bs.modal', resetRadioButtons);
         };
 
+        const redirectToUserEdit = () => {
+            const versionNo = form.querySelector('#user_edit_version_info_version_no').value;
+            const language = btns.find(btn => btn.checked).value;
+
+            window.location.replace(global.Routing.generate('ez_user_update', { contentId, versionNo, language }));
```

Comment:
I don't like the `replace` here. The last item from the history will be replaced so the user will not be able to click back button in the browser (he will be redirected to the previous page).
I think better is just: 'window.location.href = url'